### DISCUSSION TITLE
Unable to install BIRT Plugin in Eclipse Kepler 4.3.1 in Mac

### DISCUSSION THREAD

**Discussion by VL170304** [eclipse-birt/birt#1664] [Q&A]
I have downloaded eclipse-standard-kepler-SR1-macosx-cocoa-x86_64 and installed Eclipse Kepler 4.3.1 and while trying to install the Kepler plugin the below error comes up. Please note that internet explorer is working.
<img width="1432" alt="image" src="https://github.com/eclipse-birt/birt/assets/168845347/4268c7f1-5212-4528-bc86-e959587dc76c">
<img width="1177" alt="image" src="https://github.com/eclipse-birt/birt/assets/168845347/18c712b1-b9b4-4744-9312-7235ed654973">

**Comment by merks** [comment]
It seems your very old version of Java doesn't recognize the root certificate of the certificate used by the server:

https://stackoverflow.com/questions/12961570/sslpeerunverifiedexception-peer-not-authenticated
